src/modules/70-pipeline/components/PipelineInputSetForm/__tests__/mocks.tsx

```diff
@@ -687,3 +689,11 @@ export const getCICodebaseParallelTemplateProps = ({ formik }: { formik: any }):
     orgIdentifier: 'default'
   }
 })
+
+export function MockedIACMMFE({ children }: { children: ReactElement }): ReactElement {
```

Comment:
Nothing really wrong here, but `PropsWithChildren` is a great type to use for things like this.
```suggestion
export function MockedIACMMFE({ children }: PropsWithChildren<unknown>): ReactElement {
```

Even better when you can use an arrow function and `FC` (some modules don't allow arrow functions for some really silly reason).
```suggestion
export const MockedIACMMFE: FC<PropsWithChildren<unknown>> = ({ children }) => {
```

Reply:
TIL!

Reply:
If you're wondering about the `unknown` bit, it just means that we're only specifying the `children` prop. If you had other props, you can put them in there instead.

```
interface MyComponentProps {
  prop1: string
  prop2: boolean
}

const MyComponent:FC<PropsWithChildren<MyComponentProps>> = ({ children, prop1, prop2 }) => {
  // ...
}
```